app/controllers/api/alert_definition_profiles_controller.rb

```diff
@@ -21,5 +21,68 @@ def edit_resource(type, id = nil, data = {})
         raise BadRequestError, "Failed to update alert definition profile - #{err}"
       end
     end
+
+    def get_assignments_resource(_type, id, _data)
```

Comment:
Instead of using this, have you considered using [set_additional_attributes](https://github.com/ManageIQ/manageiq-api/blob/9a2767c050bf7bb943b0768ad650114c735ff445/app/controllers/api/alert_definitions_controller.rb#L5)? I just tested it out for `alert_definition_profiles` by setting `get_assigned_to` as an additional attribute, with a method `fetch_alert_definition_profiles_get_assigned_tos(resource)` and it worked well. You may want to consider restricting it to just `:show`

Reply:
I didn't know it's an option, I'll try using it and see if it works